bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/configurable/providers/cli/CliProvider.java

```diff
@@ -230,4 +234,18 @@ private CliArg getCliArg(Module module, VariableKey variableKey) {
         }
         return new CliArg(key, cliVarKeyValueMap.get(key));
     }
+
+    private static int stringToByte(String value) {
+        int byteValue = Integer.parseInt(value);
+        if (!isByteLiteral(byteValue)) {
+            throw ErrorCreator.createError(StringUtils.fromString(String
+                    .format(INVALID_INT_TO_BYTE_CONVERSION, value)));
+        }
+        return byteValue;
+    }
+
+    static boolean isByteLiteral(long longValue) {
```

Comment:
Range check we should able to use RuntimeUtils.isByteLiteral

Reply:
used accordingly